basic_features/basic_save_game_info.py

```diff
@@ -31,81 +49,166 @@ def allFiles(self) -> list[str]:
         return [self.getFilepath()]
 
 
+def get_filedate_metadata(p: Path, save: mobase.ISaveGame) -> Mapping[str, str]:
+    """Returns saves file date as the metadata for `BasicGameSaveGameInfoWidget`."""
+    return {"File Date:": format_date(save.getCreationTime())}
+
+
 class BasicGameSaveGameInfoWidget(mobase.ISaveGameInfoWidget):
+    _preview_width = 320
+
     def __init__(
         self,
         parent: QWidget | None,
         get_preview: Callable[
```

Comment:
Maybe change this to also get the savegame to be consistent between `get_preview` and `get_metadata`.